projects/ycsb-workloada2.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Workload A: Update heavy workload
+runner: ycsb
+params:
+  dir: ycsb
+  args:
+    - workloads/workloada2
+    - mongodb.url={{.MONGODB_URI}}
+
+results:
+  mongodb:
+    stats:
+      pass: 3
+
```

Comment:
No `ferretdb-postgresql` as CI did not finish within a reasonable duration https://github.com/FerretDB/dance/actions/runs/13805297011/job/38614821554?pr=1189